### PR TITLE
This kills the bad officer - Disabler nerf

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Title. This basically nerfs disablers a slowdown weapon. They are not able to deal stamina damage to someone who has 75 or more stamina damage. Essentially, this makes the USP and batons have an actual use, and punishes you for spamming disablers through windows.

Also, buffed stamina damage of the USP rubber bullets to match the disabler(from 25 to 28).

## Why It's Good For The Game

Why is a rubber loaded pistol worse than a disabler

## Changelog
:cl:
balance: Rubber 9mm shots now deal 2.5 brute damage. Just a mosquito sting.
balance: Rubber 9mm now deals a biiiit more stamina damage.
balance: Killed the disabler meta. Disablers now only work up to a certain point, past 75 stamina damage will require you to use rubber bullets or the baton to down the target.
/:cl:

### CODE REVIEW

**Comment by KathrinBailey** [comment]
Are there any security players or antag players at all that even want disables nerfed?
I've seen no one complain about it nor anyone wanting them to receive yet another nerf.
You haven't given a reason to nerf disablers. The baton already has a use, it makes people drop their active hand and knocks them down. Firing disablers through windows isn't an effective deterrent to antags. Just walk away 4HEad.

**Comment by Owai-Seek** [comment]
Isn't the entire point of the "best" disabling weapon being a laser to make it so you can't hold an entire bag of spare ammo to down people with? You can just carry a ton of spare magazines for a ballistic weapon.

**Comment by BobJoga** [comment]
> Are there any security players or antag players at all that even want disables nerfed?
> I've seen no one complain about it nor anyone wanting them to receive yet another nerf.
> You haven't given a reason to nerf disablers. The baton already has a use, it makes people drop their active hand and knocks them down. Firing disablers through windows isn't an effective deterrent to antags. Just walk away 4HEad.

Reason being to give ballistic weapons an use. The USP is near useless at the moment, and JungleRat (who plays both sec and antag frequently) has already relayed to me that disablers are kind of a meme and can just be spammed, on top of pretty much ignoring atmor.

**Comment by BobJoga** [comment]
> Isn't the entire point of the "best" disabling weapon being a laser to make it so you can't hold an entire bag of spare ammo to down people with? You can just carry a ton of spare magazines for a ballistic weapon.

Fair point. But disablers are still busted because they, like i said, pretty much ignore armor and pass through windows and can easily be spammed.

**Comment by YakumoChen** [comment]
I'd say USPs are kind of a meme weapon that feels like a needless addition to the security arsenal. Isn't NT weaponry primarily focused around beam weapons?

**Comment by ForrestWick** [comment]
> I'd say USPs are kind of a meme weapon that feels like a needless addition to the security arsenal. Isn't NT weaponry primarily focused around beam weapons?

The long-term goal we've had since December has been to replace majority beam weapons with ballistics for the station security and armory. Despite this, disablers are fine as is and don't need a nerf like this. At most the ammo for guns with disablers should be lowered a tiny bit, but even then that's not even good or needed

**Comment by BobJoga** [comment]
> > I'd say USPs are kind of a meme weapon that feels like a needless addition to the security arsenal. Isn't NT weaponry primarily focused around beam weapons?
> 
> The long-term goal we've had since December has been to replace majority beam weapons with ballistics for the station security and armory. Despite this, disablers are fine as is and don't need a nerf like this. At most the ammo for guns with disablers should be lowered a tiny bit, but even then that's not even good or needed

Disablers are always the preferable altwrnative to stun someone. There is not a single moment in which the USP is more desirable and many peopld have complained about it.

**Comment by ForrestWick** [comment]
> > > I'd say USPs are kind of a meme weapon that feels like a needless addition to the security arsenal. Isn't NT weaponry primarily focused around beam weapons?
> > 
> > 
> > The long-term goal we've had since December has been to replace majority beam weapons with ballistics for the station security and armory. Despite this, disablers are fine as is and don't need a nerf like this. At most the ammo for guns with disablers should be lowered a tiny bit, but even then that's not even good or needed
> 
> Disablers are always the preferable altwrnative to stun someone. There is not a single moment in which the USP is more desirable and many peopld have complained about it.

Then buff the damn USP somehow

**Comment by KathrinBailey** [comment]
Many people haven't complained about it, just jungerat. There's no need to make the pistol, that's already a direct upgrade to laser guns once swiped with a HoS ID, a better version of the disabler. Just give it equal stamina damage to the disabler. It's already a free weapon that bypasses recharging completely.
If you want the USP to be a preferable choice, up the amount of magazines that are in the vendor to strengthen it's gimmick, don't just nerf disablers into obscurity so everyone picks your gun instead.
Alternatively, make the rubber bullets it uses printable roundstart from the seclathe if it isn't already.

**Comment by BobJoga** [comment]
> Many people haven't complained about it, just jungerat. There's no need to make the pistol, that's already a direct upgrade to laser guns once swiped with a HoS ID, a better version of the disabler. Just give it equal stamina damage to the disabler. It's already a free weapon that bypasses recharging completely.
> If you want the USP to be a preferable choice, up the amount of magazines that are in the vendor to strengthen it's gimmick, don't just nerf disablers into obscurity so everyone picks your gun instead.
> Alternatively, make the rubber bullets it uses printable roundstart from the seclathe if it isn't already.

Muhreen has complained about it too and some other people i dont remember becausey brain has the mental capacity of a 5 year old. It's not a direct upgrade to the disabler, beczuse swiping the ID only serves to use lethals, which i will boldly assume isn't going to be used against petty criminals. The rubber bullets are already printable.

> Then buff the damn USP somehow
I have two options to buff the USP, both equally asinine and terrible.
1. I make the rubber bullets penetrate armor. That is fucking stupid and requires no further explanation on why.
2. I make the rubber bullets deal more damage than the disabler. That is fucking stupid and makes the detective's revolver actually need a buff to justify it's existence when compared to the USP.

There are some other equally terrible options which i could do too:
- Giving the USP a bigger magazine despite it already holding 15 shots
- Giving the USP burst fire.
- Making the USP use .45 which is just a buff all around.
- Giving security officers two USPs roundstart so they can akimbo them and pretend to be John Wick.
- Making the rubber bullets deal confusion.
- Making the rubber bullets cause the target to hallucinate.
- Making rubber bullets cause brain traumas when hitting the head.
- Making the USP have auto aim.
- Making the USP fire 12 gauge.
- Swap out the USP for a FN Fal.
- Completely bring back stun combat.
- Blackmail exadv1 into supporting this PR.

**Comment by Moostard** [comment]
> Many people haven't complained about it, just jungerat. There's no need to make the pistol, that's already a direct upgrade to laser guns once swiped with a HoS ID, a better version of the disabler. Just give it equal stamina damage to the disabler. It's already a free weapon that bypasses recharging completely.
> If you want the USP to be a preferable choice, up the amount of magazines that are in the vendor to strengthen it's gimmick, don't just nerf disablers into obscurity so everyone picks your gun instead.
> Alternatively, make the rubber bullets it uses printable roundstart from the seclathe if it isn't already.

I play quite a lot of antag and sometimes sec on Skyrat and other servers.
Disablers are a meme and it is true.
One-two hits from a disabler slows you down which makes you eat another disabler shots.
You cant run from them, once you get shot enough you are in a massive stamina crit which makes you unable to interact with anything around you for a long time.
If you get shot with it and fall down, its game over.
A baton makes the officer come close to you rather than spam a gun from safe distance and casually put the cuffs on as there is no way you get away from that.
People that want to run around for longer time than few minutes when going loud probably have martial arts or desword or something that blocks projectiles as disablers are just too powerful(and the fact that you can have top armor there is and you still get downed so quickly).

**Comment by RatFromTheJungle** [comment]
@KathrinBailey i would say that's blatantly incorrect, disablers are objectively the better choice due to Kevinz wonderful idea that they should ignore armor, meaning anyone from Death-squad to a lowly assistant in a cardborg suit goes down in 5-6 shots, while also being allowed to shoot through windows, having a taser shot that makes them competitively better then the USP at everything fight-wise, and having the fantastic reality that is has almost as many shots as the USP, two magazines of USP ammo only giving you 5 more shots then a fully charged disabler.

it's not the the USP isn't good, it's that the Disabler is leagues above it, at nearly everything, and the 'just give them more ammo' idea doesn't work, because most sec that i've seen already take both anyway.

the trade off should be 
USP has to deal with armor, but can fully down, aswell as not having the tasershot
the disabler cannot fully down but also doesn't have to respect armor, but has the tasershot and can shoot through windows

**Comment by KathrinBailey** [comment]
Alright, first of all.

Do your research before making PRs like this. The fact that you **didn't even bother** to look at disabler code shows how much thought you put into this.
image
It isn't ignoring armour.
image
Where is the AP variable?
image
Well it should be in the projectile, but maybe there's some old pre-AP code or something it uses? **Nope.**
image
Well there's nothing in the gun itself dealing with armour penetration.

This is all because the disabler doesn't ignore armour. The disabler does energy damage.
In fact, if I break open any security locker, like the one in departures, arrivals, engi and sci checkpoint, the one in delta maint, **I have more fucking armour against disablers than I do against 9mm rubber bullets.** Vests have 40 energy armour and 30 bullet armour. 

So over a span of five to six seconds, how long does it take to reach stamcrit?
So with a sec vest, fairly easily obtainable armour in lockers all over the map, it's ten shots to stamcrit over 5-6 seconds. Nine with combat mode.
Without a vest, it's seven shots to stamcrit over 5-6 seconds. Six with combat mode.

I looked at what Moostard said and I'm wondering if he is playing on the same codebase.
Here's a speed test.
How long does it take to run 20 tiles?
0 stamina damage: 2.5 seconds
70 stamina damage: 3 seconds
100 stamina damage: 3.5 seconds
139 stamina damage: 3.9 seconds
After taking four disabler shots, the slowdown is extremely minor. It's barely really anything at all, it's not going to slow you down enough to be open to taking more disabler shots anymore than having no stamina damage.

Now if you want some actual nerfs for disablers:
Raise their ecost in projectiles from 40 to 50. This will give you 20 shots instead of 25.
Add fire_delay = 5 to give it about the same rate of fire as the USP.

Raising the USP damage to the same as the disabler is already more than enough.
You have 15 shots with the USP, which in three seconds, you can fully 'recharge' it to 15 again by swapping the magazine. This is opposed to having your disable unusable after firing 25 until you return to security and use a charging station and wait the painful amount of time it takes to recharge the cell back to 1000 charge.
I can print full magazines at an autolathe. A normal one. Roundstart at cargo.
I can also just carry ammo boxes that I can get from autolathes and in five seconds, I can use the ammo box to refill the magazine and put it back into the gun if I don't want to carry magazines.
**Yes, you can just fill a box with seven 9mm rubber boxes and in a single box (the grey ones that you can get in the loadout menu, the one that your internals come in-), you can carry around 210 bullets, or 105 bullets if you fill a box with magazines. Going to a maximum capacity of 225 rubber bullets that takes a small weight and a medium weight in your backpack. (the pistol and a box of 7 ammo boxes).**

tl;dr
do your research before making balance PRs
increase 9mm rubber stam damage to that of a disabler + add fire delay = 5 to the disabler **OR** up the e-cost to 50 (from 40)

**Comment by Trilbyspaceclone** [comment]
As a reminder, MASON Rig in the OG verson had 500 engery armor aka 95 do to caps
This was a problem do to making both the taser and the disabler deal next to 0 damage do to being so highly armored.
Disablers and yes tasers too deal with armor. Unless Skyrats have made changes to how they work, they DO restpect armor vaules. Its why you dont see anyone using a disabler on a nukie these days.

**Comment by KathrinBailey** [comment]
> Disablers and yes tasers too deal with armor. Unless Skyrats have made changes to how they work, they DO restpect armor vaules. Its why you dont see anyone using a disabler on a nukie these days.
I tested this as well.
Combat mode syndie hardsuit that is available to traitors.
40 energy armour 50 bullet armour.

**Comment by Trilbyspaceclone** [comment]
Theirs a huge reason why Engery armor in **all but 3 clothing items** is lower then lasers.
The issue here with bullets so that /Everything has from 20 to 70% bullet armor/
To make "rubber bullet viable" - heres a pro tip dont fucken shoot lims, Lims are horrable to shoot at in gernal do to having their own hp and not that well of a bleed over
would be for them to have AP or a do after affect that DOES bypass armor.

**Comment by KathrinBailey** [comment]
Oh we're also just completely ignoring that the disabler and stunbaton is hard countered by EMPs.

**Comment by RatFromTheJungle** [comment]
https://cdn.discordapp.com/attachments/640760727489085450/718751463001555084/munke.mp4

here is a fantastic comparison of the stark difference in energy 'armor'

standard sec vest: 8 shots, 3 up from the 5 to take down a normal crewman meanwhile death squad armor takes 11, 3 up from the 8 previously mentioned, I know, it's blisteringly large, especially considering the disablers 25 round charge

and if you want to talk ammunition capacity, nothing is stopping them from carrying four disablers, giving them a total of 100 shots, or eight deathsquad officers worth of health, or even then two disablers, and having enough to take down about eight crewman, but its a good thing they don't immediately bring the people they fought to the place with all their weapon chargers, right?

**Comment by Trilbyspaceclone** [comment]
1 EMP makes all 4 of the disablers useless, and when we are talking about viablity its much easyer to get 7 mags and 2 UPS and duel wield them to deal more stam damage per second then 2 disablers on agv do to the start differents in the armor vaules.

**Comment by RatFromTheJungle** [comment]
How often do you see an EMP, should antags be forced to bring an EMP, when they're supposed to be story makers?

**Comment by KathrinBailey** [comment]
> How often do you see an EMP, should antags be forced to bring an EMP, when they're supposed to be story makers?

No one is saying this so stop acting like we're saying you should be required to bring EMPs.
EMPs are available for very low TC cost and antags like ninja get multiple powerful EMPs.
It's an easy way to counter disablers and stunbatons as well as disable headsets It's also going to randomly bolt doors.

**Comment by YakumoChen** [comment]
USP pistols and magazines can be gotten straight from the security vendor. You can carry an absurd amount of reloads for your taser equivalent weapon (with a built in lethal switch if you ask the HoS)

Ask an HoS how often they let officers carry around four disablers, given you need to use the armoury or cargo to get more

(Hint: never)

**Comment by YakumoChen** [comment]
Furthermore if USPs deal brute damage and encourage / force security to use them via mechanics you will have the silicons making your life utter fuxking hell as security, literally constantly, because muh harm

**Comment by BobJoga** [comment]
> Furthermore if USPs deal brute damage and encourage / force security to use them via mechanics you will have the silicons making your life utter fuxking hell as security, literally constantly, because muh harm

It's 2.5 brute per shot. At the very worst it may amount to 10 damage.

**Comment by BobJoga** [comment]
I got the memo. I'll make a better PR which no one will complain about, i'm *sure*.
